demos/JWST/S4_template.ecf

```diff
@@ -9,6 +9,9 @@ wave_min        1.5         # Minimum wavelength. Set to None to use the shortes
 wave_max        4.5         # Maximum wavelength. Set to None to use the longest extracted wavelength from Stage 3.
 allapers        False       # Run S4 on all of the apertures considered in S3? Otherwise will use newest output in the inputdir
 
+# Mannualy mask pixel columns by index number
+# mask_columns    []
```

Comment:
When we update the demos ECFs, we should also update the ECFs in docs/media. I see that the S1 file in that folder also didn't get updated when the GLBS was first added, so if you could update that ECF at the same time that'd be great.

Also, please document this change in the docs/source/ecf.rst file

Reply:
I'm updating the docs/media ECFs and ecf.rst file as part of the Bias Correction PR (#507).